Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -113,7 +113,7 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: ClothingEyesBase
+  parent: [ClothingEyesBase, BaseSyndicateContraband]
```

Comment:
The outlaw glasses are the ones with fake brows and nose. I am not sure if making them contraband is a good idea. They are a joke item, and this only makes them "bad" to ever buy. Technically, stuff like syndie pyjamas is considered contraband, so there's a precedent, but still.

Reply:
My thought with this was to mirror the change to cat ears that was made in #30930 and merged in. Those are also a joke item.